### ISSUE TITLE
(Switch) Yoshi Story Background issue regression

### ISSUE THREAD

**Issue by HappehLemons** [libretro/mupen64plus-libretro-nx#232]
#97

Happening in 363c47a on switch. bgMode set to OnePiece.

Doesn't happen with stripped, but game is unplayable with stripped as FPS drop to about 1 frame per second when moving.

Issue is not present in Lakkas build with matching settings, but I can't find a build version on the core.

**Comment by m4xw** [comment]
I mean, this is a GLideN64 issue, not a -next Issue.
You might wanna confirm on standalone and make a report for gonetz.

**Comment by HappehLemons** [comment]
Doesn't happen in stand alone. I'll make a report there. Thanks.

**Comment by m4xw** [comment]
If it doesn't happen in standalone then don't make a report, lol.
Do you use some kind of texture enhancements?

**Comment by m4xw** [comment]
Also show me all your options

**Comment by HappehLemons** [comment]
No custom textures or enhancements.

mupen64plus-43screensize = "640x420"
mupen64plus-alt-map = "False"
mupen64plus-aspect = "4:3"
mupen64plus-astick-deadzone = "15"
mupen64plus-astick-sensitivity = "100"
mupen64plus-BackgroundMode = "OnePiece"
mupen64plus-BilinearMode = "standard"
mupen64plus-CorrectTexrectCoords = "Off"
mupen64plus-CountPerOp = "0"
mupen64plus-cpucore = "dynamic_recompiler"
mupen64plus-d-cbutton = "C3"
mupen64plus-DitheringPattern = "False"
mupen64plus-DitheringQuantization = "False"
mupen64plus-EnableCopyAuxToRDRAM = "False"
mupen64plus-EnableCopyColorToRDRAM = "Async"
mupen64plus-EnableCopyDepthToRDRAM = "Software"
mupen64plus-EnableEnhancedHighResStorage = "False"
mupen64plus-EnableEnhancedTextureStorage = "False"
mupen64plus-EnableFBEmulation = "True"
mupen64plus-EnableFragmentDepthWrite = "True"
mupen64plus-EnableHWLighting = "False"
mupen64plus-EnableLegacyBlending = "False"
mupen64plus-EnableLODEmulation = "True"
mupen64plus-EnableN64DepthCompare = "False"
mupen64plus-EnableNativeResFactor = "0"
mupen64plus-EnableNativeResTexrects = "Disabled"
mupen64plus-EnableOverscan = "Enabled"
mupen64plus-EnableShadersStorage = "True"
mupen64plus-EnableTextureCache = "True"
mupen64plus-ForceDisableExtraMem = "False"
mupen64plus-FrameDuping = "True"
mupen64plus-Framerate = "Original"
mupen64plus-FXAA = "0"
mupen64plus-HybridFilter = "True"
mupen64plus-IgnoreTLBExceptions = "False"
mupen64plus-l-cbutton = "C2"
mupen64plus-MaxTxCacheSize = "4000"
mupen64plus-MultiSampling = "0"
mupen64plus-OverscanBottom = "0"
mupen64plus-OverscanLeft = "0"
mupen64plus-OverscanRight = "0"
mupen64plus-OverscanTop = "0"
mupen64plus-pak1 = "memory"
mupen64plus-pak2 = "none"
mupen64plus-pak3 = "none"
mupen64plus-pak4 = "none"
mupen64plus-r-cbutton = "C1"
mupen64plus-rdp-plugin = "gliden64"
mupen64plus-RDRAMImageDitheringMode = "False"
mupen64plus-rsp-plugin = "hle"
mupen64plus-ThreadedRenderer = "False"
mupen64plus-txCacheCompression = "True"
mupen64plus-txEnhancementMode = "None"
mupen64plus-txFilterIgnoreBG = "True"
mupen64plus-txFilterMode = "None"
mupen64plus-txHiresEnable = "False"
mupen64plus-txHiresFullAlphaChannel = "False"
mupen64plus-u-cbutton = "C4"
mupen64plus-virefresh = "Auto"

**Comment by m4xw** [comment]
There have been a few upstream fixes, will pull them in over the next days

**Comment by HappehLemons** [comment]
Happy to retest, just let me know.

**Comment by m4xw** [comment]
Closing in favour of https://github.com/gonetz/GLideN64/issues/2304

**Comment by m4xw** [comment]
https://git.libretro.com/libretro/mupen64plus-libretro-nx/-/pipelines/4122/builds
If everything works out I will push it later

**Comment by dagreatbuzzsaw** [comment]
I'm sorry for being a bother and bringing this up though this reads that the issue is closed but has it really been fixed???  I mean, I downloaded the latest Retroarch and updated the mupen64plus-next core, and I am experiencing the same problems mentioned here at the top of this thread.  I looked everywhere for a fix to this and I am not getting any straight answers.  Can anyone help?

Again, sorry.
